content/applications/general/payment_acquirers/adyen.rst

```diff
@@ -99,5 +99,29 @@ where payments will be made (the URLs of the servers hosting your Odoo instances
    :align: center
    :alt: Allows payments originated from a specific domain.
 
+Capture the payment after the delivery
+--------------------------------------
+
+If you wish to manually capture an amount instead of having an immediate capture, Adyen allows it.
+
+You have to enable it both on Odoo, as explained :ref:`here <payment_acquirers/capture_amount>`,
+and on Adyen.
+
+To do so, on your Adyen Merchant Account, go to :menuselection:`Account --> Settings`, and set the
+**Capture Delay** to **manual**.
+
+.. image:: media/adyen_capture_delay.png
```

Comment:
Media file names use hyphens and are in a distinct media folder, named as the doc's filename.
Please refer to the [Content guidelines: images](https://www.odoo.com/documentation/14.0/contributing/documentation/content_guidelines.html#media-files) for more information.
If needed, you can also move previous images to that new folder

Reply:
Since I messed it up when I rewrote the payment acquirers documentation (this PR was written before, seemed like I merged unused file... my bad...), this image is already there. I could change just this image now, but we'll change all the images at once in another task, since this need to be done for all of it. It makes more sense than making a single change now.

Reply:
Well, indeed apologies, I hadn't realized it isn't a new image. You're 100% right!